include/benchmark/benchmark.h

```diff
@@ -888,7 +932,15 @@ class Benchmark {
   // SetIterationTime(seconds) to report the measured time, which will be used
   // to control how many iterations are run, and in the printing of items/second
   // or MB/second values.
+  // The second overload creates a manual time which measures in some arbitrary
+  // value set by the user. The Cost Function returns the "cost" of a single
```

Comment:
But that is not really what the `Cost Function` does, isn't it?
It converts the given custom measurement to the seconds.

Reply:
Hmm, i think i mis-commented.
```
 The Cost Function returns the "cost" of a single
 benchmark iteration's in seconds, to be used to determine the cost
 of a single run of the benchmark max iteration estimation and so on.
```
My question was about the word `single`.
We don't have any such guarantees, do we?
I think it will always end up being called for whatever the timer has counted,
without division by iteration count, i think.